js/src/modals/FundAccount/AwaitingDepositStep/awaitingDepositStep.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+
+import Value from '../Value';
+
+import styles from '../fundAccount.css';
+
+export default class AwaitingDepositStep extends Component {
+  static propTypes = {
+    coinSymbol: PropTypes.string.isRequired,
+    depositAddress: PropTypes.string,
+    price: PropTypes.shape({
+      rate: PropTypes.number.isRequired,
+      minimum: PropTypes.number.isRequired,
+      limit: PropTypes.number.isRequired
+    }).isRequired
+  }
+
+  render () {
+    const { coinSymbol, depositAddress, price } = this.props;
+    const typeSymbol = (
+      <div className={ styles.symbol }>
+        { coinSymbol }
+      </div>
+    );
+
+    if (!depositAddress) {
+      return (
+        <div className={ styles.center }>
+          <div className={ styles.busy }>
+            Awaiting confirmation of the deposit address for your { typeSymbol } funds exchange
+          </div>
+        </div>
+      );
+    }
+    return (
```

Comment:
In `AwaitingDepositStep`, `AwaitingExchangeStep`, `CompletedStep`, `ErrorStep` and `OptionsStep`, there's a lot of duplicate/boilerplatisch code. It's a lot of stuff to read through for not soo much complexity being handled; Maybe we can rduce that a little. :)

Also, but this is a matter of taste, having separate directories for these steps is bloat.